src/classes/game/game.rb

```diff
@@ -0,0 +1,10 @@
+class Game
+  def initialize(last_played_at)
```

Comment:
Kindly add a multiplayer(bool) parameter to initialize Game and be assigned to @multiplayer instance